README.rst

```diff
@@ -53,29 +53,33 @@ http://nitrate.readthedocs.org/
 How to run
 ----------
 
-There are a series of documentation for installing Nitrate in
-`Installation Guide`_. Choose one for your case. It is appreciated to catch any
-issues of the documentation, report it or fix it.
+There are a series of instructions for installing Nitrate in the
+`Installation Guide`_. Skim through the documentation and choose the installation instructions that are appropriate for your case. And please, if you identify any issues with the installation guide, kindly bring it to our attention. You can either report the issue on the github repo, or submit a PR with a fix for it.
```

Comment:
Let's wrap all lines of text at column 80 in documentation, except those lines that can't be wrapped. This would keep the text easy-to-read even if open it up and read in a text editor.

Reply:
Okay @tkdchen.

Thank you for your feedback.

I've just updated the file to keep lines of text at 80 characters max.